dark/models/tournament/tournament.py

```diff
@@ -1,17 +1,27 @@
+import random
+
 from django.db import models
 from dark.models.user.user import User
 
 
+def create_new_access_key():
+    return random.randint(1000, 9999)
+
+
 class Tournament(models.Model):
     name = models.CharField(max_length=30, blank=False)
-    creator = models.ForeignKey(User, on_delete=models.CASCADE)
+    creator = models.ForeignKey(User, on_delete=models.CASCADE, related_name='creator')
```

Comment:
Dobrze by to było przenieść na konceptualną bazę danych (żeby potem nie było wątpliwości przy rozszerzaniu funkcjonalności) (bardzo opcjonalne jak ci się chce)

Reply:
Potrzebuję bardziej szczegółowego opisu (nie bardzo rozumiem czym jest konceptualna baza danych)